examples/shader/shader_material.rs

```diff
@@ -56,6 +63,48 @@ fn setup(
     });
 }
 
+fn rotate_mesh(time: Res<Time>, mut query: Query<&mut Transform, With<Handle<Mesh>>>) {
+    let angle = std::f32::consts::TAU / 12.0;
+    let mut transform = query.single_mut();
+    transform.rotate(Quat::from_rotation_x(angle * time.delta_seconds()));
+    transform.rotate(Quat::from_rotation_y(angle * time.delta_seconds()));
+}
+
+fn cube_vertex_colors() -> Vec<[f32; 4]> {
```

Comment:
I'd prefer to not complicate this example with more things on top of what it already does. Custom vertex attributes should be a separate example.

Reply:
Cool, I'll move this to a new example.